config/site_settings.yml

```diff
@@ -1069,6 +1069,9 @@ embedding:
   embed_post_limit:
     default: 100
     hidden: true
+  embed_link_to_top:
+    default: false
+    hidden: true
```

Comment:
Curious why this is a hidden site setting?  This seems like something that could be exposed 🤔